node/node.go

```diff
@@ -33,6 +34,12 @@ const (
 	gcPercentageTime = 0.1
 )
 
+var (
+	eigenDAUIMap = map[string]string{
+		"5": "https://goerli.eigenlayer.xyz/avs/eigenda",
```

Comment:
Maybe comment it's mapping from chain ID to URL.

Reply:
done